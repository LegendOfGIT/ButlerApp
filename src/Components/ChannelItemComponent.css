.ChannelItem {
  margin-bottom: 20px;
  max-width:350px;
  min-height: 350px;
  padding: 15px 0 12px 15px;
  text-align: center;
}

.ChannelItemHeader {
  padding: 15px;
  text-align: center;
}
