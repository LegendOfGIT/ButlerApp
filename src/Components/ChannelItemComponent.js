import React from 'react';

const ChannelItemComponent = () => {
    return (
        <div className="ChannelItem">
            <div>Title</div>
            <div>Information-Header</div>
            <div>Description</div>
        </div>
    );
};

export default ChannelItemComponent;