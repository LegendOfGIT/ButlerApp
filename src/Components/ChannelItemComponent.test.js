import React from 'react';
import ChannelItemComponent from './ChannelItemComponent';
import TestRenderer from 'react-test-renderer';

describe('ChannelItem Component', () => {
    it('renders without crashing', () => {
        expect(TestRenderer.create(<ChannelItemComponent/>).toJSON()).toMatchSnapshot();
    });

    it('renders information "title" when property is set', () => {
        const props = { title: 'The Simpsons in 10 Minuten auf Pro-Sieben' };
        expect(TestRenderer.create(<ChannelItemComponent {...props} />).toJSON()).toMatchSnapshot();
    });
});

