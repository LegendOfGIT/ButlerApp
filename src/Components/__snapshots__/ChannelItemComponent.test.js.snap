// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ChannelItem Component renders information "title" when property is set 1`] = `
<div
  className="ChannelItem"
>
  <div>
    Title
  </div>
  <div>
    Information-Header
  </div>
  <div>
    Description
  </div>
</div>
`;

exports[`ChannelItem Component renders without crashing 1`] = `
<div
  className="ChannelItem"
>
  <div>
    Title
  </div>
  <div>
    Information-Header
  </div>
  <div>
    Description
  </div>
</div>
`;
